### PR TITLE
Documentation - properly escaped backtick in "Class`1"

### PR DESCRIPTION
**Pull request by RaYell**
The original had incorrectly escaped backtick which was not shown ourside of edit form. I've updated the code to properly print it.

### CODE REVIEW

File: documentation/SA1649.md [hunk 1]
```diff
@@ -18,7 +18,7 @@
 ## Cause
 
 The file name of a C# code file does not match the first type declared in the file. For generics that are defined as
-`Class1<T>` the name of the file needs to be `Class1{T}.cs` or `Class1``1.cs` depending on the `fileNamingConvention`
+`Class1<T>` the name of the file needs to be `Class1{T}.cs` or `Class1```1.cs` depending on the `fileNamingConvention`
```
Comment (sharwell) [inline documentation/SA1649.md]: :question: Does it work to use this instead?

    ``Class1`1.cs``

Testing: ``Class1`1.cs``

Comment (RaYell) [inline documentation/SA1649.md]: Yeah, that work as well, and is probably even more readable when you edit. I've updated the PR to use this form.

**Comment by pdelvo** [comment]
:+1:

**Comment by sharwell** [comment]
:+1:

**Pull request merged**